style/app.scss

```diff
@@ -233,10 +233,51 @@ a:active
   color           : #4c004c;
 }
 
-#disclaimerDiv
+.popup-banner
 {
-  padding   : 0px 1em 0px 1em;
-  font-size : 11pt;
+  font-size        : 11pt;
+  position         : absolute;
+  background-color : #ffdd61;
+  padding          : 5px 15px 5px 15px;
+  z-index          : 1;
+}
+
+.popup-banner.dismiss
+{
+  transition : visibility 0s 0.5s, opacity 0.5s linear;
+  visibility : hidden;
+  opacity    : 0;
+}
+
+.popup-banner a
+{
+  color : #af16af;
+}
+
+.popup-banner label
+{
+  font-weight : lighter;
```

Comment:
In general, do not introduce new styles for text or other elements unless there's a good reason we need something new. In this case, you should remove this rule and the new color.